README.md

```diff
@@ -11,3 +11,14 @@ In the project directory, you can run:
 npm install
 npm run start
 ```
+
+## Add this app to your pod using SAI
+In order to add this app to your pod:
+* use [this branch of SAI](https://github.com/michielbdejong/sai-js/tree/investigating-77), and run it on localhost according to the instructions there
```

Comment:
For the people to request authorization you will need to remove the ldo-react-tutorial one you added.
You will also need to add at least one data registry using your shapetree, for example here https://github.com/janeirodigital/sai-js/tree/main/packages/css-storage-fixture/alice-home/dataRegistry

Reply:
What happens if Alice uses this app for the first time and doesn't have the data registry it needs yet?

Reply:
-> https://github.com/janeirodigital/sai-js/issues/80